src/Agent.Worker/Release/ReleaseFileSystemManager.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.Services.Agent.Util;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
+{
+    [ServiceLocator(Default = typeof(ReleaseFileSystemManager))]
+    public interface IReleaseFileSystemManager : IAgentService
```

Comment:
These functions are called from different artifacts. I had to group these somewhere so that its reusable in other places. Or may be convert them to static methods?